index.html

```diff
@@ -30,22 +29,185 @@ <h1>I’m Mohammed Glad to see you!</h1>
         <ul>
           <li>
             <a href="https://twitter.com/RebbouhElseddik"
-              ><img src="img/twitter.svg" alt=""
+              ><img src="./img/twitter.svg" alt=""
```

Comment:
You have done a good job, but a little tweak to make it juicy. I suggest you describe your `alt` attributes to provides alternative information for an image if a user for some reason cannot view it.